security/security.c

```diff
@@ -869,9 +866,19 @@ static void __init lsm_early_task(struct task_struct *task)
 
 #define call_void_hook(FUNC, ...) do {					\
 	switch (base_slot_idx.FUNC) {					\
+	default : {							\
+		struct security_hook_list *P;				\
```

Comment:
Let's have a proper name for P like `__hook_list` (or a more consistent name if it's called by something else elsewhere)